templates/InventoryCollector.yml

```diff
@@ -37,12 +37,22 @@ Parameters:
     Default: cron(0 9,21 * * ? *)
     AllowedPattern: .+
 
+  EmailSender: 
+    Type: String
+    Default: ''
+    Description: Who sends an email with a report.
+
+  EmailRecipient: 
+    Type: String
+    Default: ''
+    Description: To whome send an email with a report.
```

Comment:
typo: whom
it's better to describe datatype in the description (like `The email address of recipient`), is it an email or name or other type?

Reply:
Done